packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -54,6 +54,45 @@ typedef ReorderCallback = void Function(int oldIndex, int newIndex);
 /// All [children] must have a key.
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=3fB1mxOsqJE}
+///
+/// This sample shows by dragging the user can reorder the items of the list.
+/// list of Numbers will be used to display items in listview.
+/// [List.generate]it is constructor which  is used to form a list of values.
+/// [onReorder] This parameter is compulsory. It will be called when list child
```

Comment:
"[onReorder] This parameter is compulsory. It will be called when list child is moved to a new position" -> "The [onReorder] parameter is required and will be called when a child widget is dragged to a new position."